packages/crud/src/RecordData.tsx

```diff
@@ -16,8 +12,8 @@ type RecordDataProps = {
 
   recordId: string,
   variables?: Object,
-  skip?: (Object) => boolean,
-  children: (recordData: QueryResult) => React$Node,
+  skip?: boolean,
```

Comment:
@ej9x why skip isn't a function?

Reply:
because it is in react-apollo typings
```
export interface QueryProps<TData = any, TVariables = OperationVariables> extends QueryOpts<TVariables> {
    children: (result: QueryResult<TData, TVariables>) => React.ReactNode;
    query: DocumentNode;
    displayName?: string;
    skip?: boolean;
    onCompleted?: (data: TData | {}) => void;
    onError?: (error: ApolloError) => void;
}
```

Reply:
ok, i see.